performance_regression_test.py

```diff
@@ -46,7 +46,26 @@ def __init__(self, *args):
         self._clean_email_data()
         super().__init__(*args)
 
+    @teardown_on_exception
+    @log_run_info
+    def setUp(self):
+        super().setUp()
+        if self.params.get("run_db_node_benchmarks"):
+            self.log.info("Validate node benchmarks results")
+            compare_results = self.db_cluster.get_node_benchmarks_results() or {}
+            ready_nodes = []
+            for node, items in compare_results.items():
+                for item, result in items.items():
+                    ready_nodes.append(result["is_within_margin"])
+                    if not result["is_within_margin"]:
+                        self.log.error("HW performance test on node %s has bad results for %s : %s", node, item, result)
+            if not all(ready_nodes):
```

Comment:
Let's add a new parameter that decides if to stop the test if above margin or not.

Reply:
Done